js/PivotReport.PivotTable.js

```diff
@@ -1,4 +1,4 @@
-/* globals CRM, ts */
+/* globals _, CRM, moment, ts */
```

Comment:
How this repo has access to `_` and `moment` variables, where are they loaded?

Reply:
`moment` is included in all reports ([here](https://github.com/compucorp/uk.co.compucorp.civicrm.pivotreport/blob/master/pivotreport.php#L145), `_` is provided by CiviCRM ([here](https://github.com/civicrm/civicrm-core/blob/bc2951c050c77c431259820125248bff41e2d742/CRM/Core/Resources.php#L716). `ts` is also from core.